src/components/Button.tsx

```diff
@@ -10,7 +10,8 @@ const radius: Record<Radius, number> = {
 
 export const Button = withTheme(styled.button<{ theme: DefaultTheme }>`
   display: flex;
-  font-family: "Manrope", sans-serif;
+  font-family: "${(props) => props.theme.themeConfig.font.fontFamily}",
```

Comment:
the quotation marks are making the font configuration not work

Reply:
this is correct! I must have missed that, because it simply used the global fallback of my test application, leading to the same result. It is fixed in the next PR to correctly apply the font family.